libs/features/mas/web-components/src/merch-datasource.js

```diff
@@ -52,75 +49,127 @@ const cardContent = {
         },
         ctas: { size: 'l' },
     },
+    'special-offers': {
```

Comment:
We should discuss if it makes sense to split merch-datasource.
It is the same situation as with merch-card block in milo it will keep growing with each card variant added and might become too complicated.

Reply:
and typically mapping could be held, as well as css in one given variant file

Reply:
I like this idea, files are getting too bulky and harder to maintain.

Reply:
this will be addressed post PR merge as tech-debt.